backend/backend/settings.py

```diff
@@ -124,3 +127,7 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+CORS_ORIGIN_WHITELIST = [
+     'http://localhost:3000'
```

Comment:
Du coup je me suis renseigné comment on va gérer la mise en production : parce que par exemple, quand on sera en prod, cela ne sera pas la même adresse ;) https://stackoverflow.com/questions/10664244/django-how-to-manage-development-and-production-settings